.github/workflows/linux.yml

```diff
@@ -125,3 +125,29 @@ jobs:
 
         cd "$RootDir/python/gym_jiminy/examples/reinforcement_learning/rllib"
         "${PYTHON_EXECUTABLE}" acrobot_ppo.py
+
+    - name: Python static type checker
+      if: matrix.PYTHON_VERSION == '3.8' && matrix.BUILD_TYPE == 'Release'
+      run: |
+        # Generate stubs
+        stubgen -p jiminy_py -o $RootDir/build/pypi/jiminy_py/src
+        "${PYTHON_EXECUTABLE}" "$RootDir/build_tools/stub_gen.py" \
+          -o $RootDir/build/stubs --ignore-invalid=all jiminy_py
+        cp $RootDir/build/stubs/jiminy_py-stubs/core/__init__.pyi \
+           $RootDir/build/pypi/jiminy_py/src/jiminy_py/core/core.pyi
+
+        # Re-install jiminy with stubs
+        cd "$RootDir/build"
+        make install
+
+        # mypy is not able to follow `.pth` when checking directories or files.
+        # This bug may be fixed in the future. The only workaround is to set `MYPYPATH` manually.
+        # Anyway, the stubs generated for eigenpy, hppfcl and pinocchio they are incomplete and
```

Comment:
Why do claim that here?!

Reply:
If there are errors, it would be kind to fix them and contribute to the original source you took inspiration from (https://github.com/jcarpent/pybind11-stubgen). I guess the use of open-source projects should go in both ways.

Reply:
> Why do claim that here?!

Just to remember for later why I'm not enabling stub generation on these projects for now. Nothing more.  So that I can  remember there is work to do upstream to improve the situation for these projects.